app/workers/open_data_publications_worker.rb

```diff
@@ -0,0 +1,75 @@
+require 'csv'
+
+class OpenDataPublicationsWorker
+  include Sidekiq::Worker
+
+  def perform(*args)
+    file = File.join(ENV['OPENDATA_CSV_PATH'],
+                     "opendata_publications_#{Time.now.strftime('%Y-%m-%d_%H-%M-%S')}.csv")
+    publications = NycGovernmentPublication.where('suppressed_bsi': 'false')
+    header = ['Title',
+              'Sub-Title',
+              'Agency',
+              'Required Report Name',
+              'Additional Creators',
+              'Subject',
+              'Description',
+              'Date Published',
+              'Report Type',
+              'Languages',
+              'Associated Year - Fiscal',
+              'Associated Year - Calendar',
+              'Associated Borough',
+              'Associate School District',
+              'Associated Community Board District',
+              'Associated Place',
+              'Filename',
+              'Last Modified']
```

Comment:
Should we have all fields enclosed in double quotes? That way spaces don't mess things up when loading this into the ETL pipeline.